projects/core/src/auth/user-auth/services/auth-http-header.service.ts

```diff
@@ -158,6 +159,14 @@ export class AuthHttpHeaderService implements OnDestroy {
     return url.includes(this.occEndpoints.getBaseUrl());
   }
 
+  protected isBaseSitesRequest(request: HttpRequest<any>): boolean {
+    return request.url.includes(
+      this.occEndpoints.getRawEndpointValue('baseSites')
```

Comment:
`baseSites` or `basesites`? or `/basesites`?

Reply:
it needs to be 'baseSites' to match the property in default-occ-site-context-config.ts -> backend.occ.endpoint.baseSites. It is the way getRawEndpointValue fct works, it picks property value of backend.occ.endpoint[baseSites]. so fails if 'basesites' or '/basesites' are used